config/helm/chart/default/values.yaml

```diff
@@ -21,7 +21,13 @@ installCRD: false
 
 operator:
   nodeSelector: {}
-  tolerations: []
+  tolerations:
```

Comment:
these tolerations should be unconfigurable and directly set in the deployment/daemonset, so tolerations that are set here are just added to the default tolerations and don't overwrite them.

Reply:
fixed